app/models/miq_worker/runner.rb

```diff
@@ -130,6 +133,11 @@ def start
     end
   end
 
+  def recover_from_provider_unreachable
+    sleep 2.minutes
```

Comment:
Ah, I *think* @matthewd might be asking whether the worker management code will kill this worker b/c it didn't heartbeat back to the server and spin up a new one...

hrmmm... @Fryguy?  Thoughts on this?  I'm thinking that's fairly likely that the worker will be killed if it's sleeping for too long.  But, I can't remember the timeout.